logos/logos-workernode/logos_worker_node/vllm_process.py

```
@@ -602,11 +696,17 @@ def _build_cmd(self, lane_config: LaneConfig) -> list[str]:
         vllm_prefix = self._resolve_vllm_binary(vc.vllm_binary)
         cmd = [
-            *vllm_prefix, "serve", lane_config.model,
-            "--host", "0.0.0.0",
-            "--port", str(self.port),
-            "--tensor-parallel-size", str(vc.tensor_parallel_size),
-            "--dtype", vc.dtype,
+            *vllm_prefix,
+            "serve",
+            lane_config.model,
+            "--host",
+            "0.0.0.0",
```

Ruff (0.15.12)

[error] Possible binding to all interfaces

(S104)